### PR TITLE
Contributor

### PR DESCRIPTION
**Pull request by celomary**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -83,18 +83,18 @@ export default function Home({ users, categories }) {
         <section className="container mx-auto px-5 pt-10">
           <h1 className="my-4 text-5xl font-bold leading-tight">Got a technical question?</h1>
           <p className="mb-8 text-3xl leading-normal">Ask our tech twitter volunteers!</p>
-          <div className="flex flex-wrap space-x-2">
```
Comment (codewithvoid) [inline src/pages/index.js:86]: can you pull from main branch..this file shouldn't be updated

Comment (codewithvoid) [reply src/pages/index.js:86]: same with package-lock.json

File: data/users/celomary.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Mohamed El omary",
+  "bio": "fontend developer | software developer",
+  "image": "https://avatars.githubusercontent.com/u/59455493?v=4",
+  "categories": [
+    {
+      "slug": "web-development | programming | software-development",
```
Comment (codewithvoid) [inline data/users/celomary.json:7]: you have to pick one of the slug per category

```json
  "categories": [
    {
      "slug": "web-development",
      "topics": [
        "ReactJS",
        "Redux"
      ]
    },
    {
      "slug": "programming",
      "topics": [
        "Python",
        "JavaScript"
      ]
    }
  ],
```

File: data/users/celomary.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Mohamed El omary",
+  "bio": "fontend developer | software developer",
```
Comment (codewithvoid) [inline data/users/celomary.json:3]: pick 1 title here

**Comment by codewithvoid** [comment]
Thank you for volunteering @celomary. Added comments.

File: data/users/celomary.json (hunk 3 above)
Comment (celomary) [reply data/users/celomary.json:3]: okay

**Pull request closed without merging**